frontend/src/components/YesNoSelection.tsx

```diff
@@ -86,10 +87,10 @@ const YesNoSelection: FunctionComponent<YesNoSelectionProps> = ({
           top={20}
           check={
             selected === 0 ? (
-              correctAnswer ? (
-                <CheckCircleFilled style={{ color: CORAL }} />
-              ) : (
+              !correctAnswer ? (
```

Comment:
Any chance we can flip this around so it is like `correctAnswer ? ... : ...`? I think that is more readable since the true case is first.

Reply:
okie